packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/general-informations/edit/EditVdcDescription.page.tsx

```diff
@@ -18,7 +18,7 @@ export default function EditVdcDescription() {
   const { addSuccess } = useMessageContext();
   const { id, vdcId } = useParams();
   const { data: vcdDatacentre } = useVcdDatacentre(id, vdcId);
-  const { updateDetails, error, isError } = useUpdateVdcDetails({
+  const { updateDetails, error, isError, isPending } = useUpdateVdcDetails({
```

Comment:
What about isLoading ?

Reply:
same answer as above, `isPending` is the boolean representing if the mutation is currently executing (equivalent to `isLoading` of a query)